app/Http/Controllers/ApiController.php

```diff
@@ -21,6 +24,24 @@ public function __construct(private readonly Request $request)
         $this->initPermissions();
     }
 
+    public function getFacilityOrFail(): Facility
+    {
+        $permissionObject = $this->getPermissionObject();
+        if ($permissionObject->facility) {
+            return $permissionObject->facility;
+        }
+        throw FatalExceptionFactory::unexpected();
+    }
+
+    public function getUserOrFail(): User
```

Comment:
będzie w przyszłości więcej tego typu metod tutaj?

Reply:
myślę, że nie, chodzi mi o to, żeby to była metoda która mi wygodnie zwraca w kontrolerze User,

a nie jakieś ?User, Authenticable czy inny dziwny typ